specification.json

```diff
@@ -263,6 +285,13 @@
                 }
             ]
         },
+        {
+            "id": "Requirement 2.2.9",
+            "machine_id": "requirement_2_2_9",
+            "content": "The `provider` SHOULD populate the `resolution details` structure's `flag metadata` field.  `flag metadata` MUST be a structure supporting the definition of arbitrary properties, with keys of type `string`, and values of type `boolean | string | number`.",
+            "RFC 2119 keyword": "MUST",
```

Comment:
Having 2 requirements in a single clause doesn't work well with the parser (you can see we only detected the first 2119 keyword.

See: https://github.com/open-feature/spec/pull/169/files/affd2ada4df2eac19efc16347fea7eccedc6cd92#r1067503620